opte/src/engine/packet.rs

```diff
@@ -1396,7 +1396,19 @@ impl Packet<Parsed> {
             }
 
             Some(IpMeta::Ip6(ip6)) => {
-                ip6.pay_len = (new_pkt_len - pkt_offset) as u16;
+                // IPv6 Payload Length field is defined in RFC 2640 section 3
+                // as:
+                //
+                // > Length of the IPv6 payloaed, i.e., the rest of the packet
```

Comment:
```suggestion
                // > Length of the IPv6 payload, i.e., the rest of the packet
```

Reply:
Nice catch thanks. I wrote it while reading it, to make sure I _understood the damn thing_.